tests/bluetooth/host/id/bt_id_set_adv_own_addr/prj.conf

```diff
@@ -7,3 +7,6 @@ CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
 CONFIG_ASSERT_VERBOSE=y
 CONFIG_ASSERT_ON_ERRORS=y
+
+CONFIG_NO_OPTIMIZATIONS=y
+CONFIG_LTO=n
```

Comment:
I'd at least add a comment above these with a short explanation, so that one doesn't have to go full-on Sherlock Holmes and track down the explanation from the commit that introduced them.

Reply:
Can @jori-nordic send it as a follow-up? This is blocking all development right now.

Reply:
> Can @jori-nordic send it as a follow-up? This is blocking all development right now.

That's ok with me, assuming the CI tests pass (they're still running when I write this)